clusterloader2/testing/density/config.yaml

```diff
@@ -58,23 +50,14 @@ steps:
       # timeout = (totalPodsRounded /20)s + 3m
       timeout: {{DivideInt $totalPodsRounded 20 | AddInt 180}}s
 - measurements:
-  - Identifier: PhaseTimer
-    Method: Timer
-    Params:
-      label: CreateSaturationPods
-      action: stop
-  - Identifier: PhaseTimer
-    Method: Timer
-    Params:
-      label: CreateLatencyPods
-      action: start
   - Identifier: PodStartupLatency
     Method: PodStartupLatency
     Params:
       action: start
       labelSelector: kind = latencyPod
 # Create latency pods
-- phases:
+- name: Create latency pods
```

Comment:
This one is also tricky - creation will take exactly: #pods / 5 - we perform 5qps :)